### PR TITLE
Update and rename geetest.xml to geetest.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

**Comment by ivysrono** [comment]
@gloomy-ghost 这个很急啦，杀伤很大

File: src/chrome/content/rules/geetest.com.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--
+	Broken:	
+	( test in http://bbs.kafan.cn/member.php?mod=logging&action=login or http://www.libreofficechina.org/member.php?mod=lor )
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/geetest.com.xml]: image

`api` 和 `static` 都跳转了也可以通过啊

Comment (ivysrono) [inline src/chrome/content/rules/geetest.com.xml]: 囧爆了，是 https://github.com/EFForg/https-everywhere/pull/7753/commits/b67774297969381b9abd4d1822f83adbebe9c368 的问题……

**Comment by gloomy-ghost** [comment]
1. The CAPTCHA passes when api.geetest.com and static.geetest.com are redirected.
https://github.com/EFForg/https-everywhere/pull/7854#discussion_r94370472

2. Still getting `refused` for https://fullpage.geetest.com
image

3. https://account.geetest.com is the same with http://account.geetest.com
https://www.diffchecker.com/YqDKBElt

Closing if #7490 covers other changes of this PR.

**Comment by ivysrono** [comment]
https://github.com/EFForg/https-everywhere/pull/7753/#discussion_r94409548
Close.

**Pull request closed without merging**